src/main/java/appeng/util/inv/AdaptorIInventory.java

```diff
@@ -293,7 +293,7 @@ private ItemStack addItems( final ItemStack itemsToAdd, final boolean modulate )
 						return null;
 					}
 				}
-				else if( Platform.isSameItemPrecise( is, left ) && is.stackSize < perOperationLimit )
+				else if( Platform.itemComparisons().isSameItem( is, left ) && is.stackSize < perOperationLimit )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method appeng.util.inv.AdaptorIInventory.addItems(ItemStack, boolean) orders expressions in a conditional in a sub optimal way [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=fb-contrib%3ASEO_SUBOPTIMAL_EXPRESSION_ORDER)